documentation/guides/record-tests/README.md

```diff
@@ -35,13 +35,13 @@ You can record a test in any local browser. TestCafe detects browsers on the loc
 
 ![Record Congiruation Menu](../../../images/guides/record-configuration-list.png)
 
-You can modify record configuration options by clicking the ![Settings button](../../../images/guides/settings-icon.png) button and using the [Record Configuration](../../user-interface/record-configuration-dialog.md) dialog.
+Click the ![Settings button](../../../images/guides/settings-icon.png) button and use the [Record Configuration](../../user-interface/record-configuration-dialog.md) dialog to modify record configuration options.
```

Comment:
@dirk-pieterse 
In my variant, I describe where you CAN modify the options if needed, it's not necessary. 
If we write "Click the .... to modify", the user can think that he/she must do it.

Reply:
'by clicking', 'using' **-ing** We should avoid using -ing. I saw this sentence and rewrote it to avoid this.

Reply:
I'll rewrite: "You can modify record configuration options in the [Record Configuration](../../user-interface/record-configuration-dialog.md) dialog. To open the dialog, click the ![Settings button](../../../images/guides/settings-icon.png) button next to the **Record configuration** menu on the toolbar."